OloEngine/src/OloEngine/Scene/Scene.cpp

```
@@ -2,6 +2,8 @@
 #include "Scene.h"
 #include "Entity.h"
 
+#include <cmath>
```

Cppcheck (2.20.0)

[information] Include file

(missingInclude)